### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by michalborkowski96**
Add helm chart

### CODE REVIEW

File: cluster-bot-chart/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          args: ["resourcebot"]
+          env:
+            - name: WIRED_TIGER_CACHE_SIZE_GB
+              value: "{{ .Values.cache_size_gb }}"
+            - name: SLACK_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.slack_token_secret.name }}
```
Comment (monester) [inline cluster-bot-chart/templates/deployment.yaml:42]: it is a name of secret resource, it should not be a parameter. As well as key name.
Also I don't see secret resource definition.

File: cluster-bot-chart/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "resourcebot.serviceAccountName" . }}
+      containers:
+        - name: resourcebot
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          args: ["resourcebot"]
```
Comment (monester) [inline cluster-bot-chart/templates/deployment.yaml:35]: this command already set in Dockerfile.

**Review by monester — CHANGES_REQUESTED** [review]

File: cluster-bot-chart/templates/deployment.yaml (hunk 2 above)
Comment (michalborkowski96) [reply cluster-bot-chart/templates/deployment.yaml:35]: It didn't work for me locally, but now I see why. The command in Dockerfile has an extra space. Apparently it's not a problem for Docker, but breaks in kubernetes.

File: cluster-bot-chart/templates/deployment.yaml (hunk 1 above)
Comment (monester) [reply cluster-bot-chart/templates/deployment.yaml:41]: when you create a chart which is using another chart as base then it is worth to specify secret and key.
However it is not crucial so lgtm.

**Review by monester — APPROVED** [review]

File: cluster-bot-chart/templates/statefulset.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: StatefulSet
```
Comment (monester) [inline cluster-bot-chart/templates/statefulset.yaml:2]: why statefulset? why you can't use deployment?

**Review by monester — APPROVED** [review]

**Pull request merged**